hm_pyhelper/sbc.py

```diff
@@ -0,0 +1,71 @@
+import os
+from enum import Enum, auto
+from collections import namedtuple
+
```

Comment:
Please add a comment to explicitly say that SBC stands for single board computer.